setup.py

```diff
@@ -275,6 +275,14 @@ def get_dynamic_setup_params():
             'config/base.yml',
             'config/module_defaults.yml',
         ],
+        'ansible_test': [
+            '*/*.*',
+            '*/*/*.*',
+            '*/*/*/*.*',
+            '*/*/*/*/*.*',
+            '*/*/*/*/*/*.*',
+            '*/*/*/*/*/*/*.*'
+        ]
```

Comment:
We should update `test/sanity/code-smell/package-data.py` to ensure things get installed.  That `package_data` looks like we could end up with missing files easily if nesting depth changes.

Reply:
I don't actually think we have a good reason to be using `package_data` at all anymore- we should be using `include_package_data=True` and setting everything up in `MANIFEST.in`. I'm updating the PR to do that (for ansible as well) and updating the sanity test to validate the installed bits for both against MANIFEST.